module/skill/QuinticWalk/data/config/Quinticwalk.yaml

```diff
@@ -0,0 +1,131 @@
+log_level: WARN
```

Comment:
These QuinticWalk yaml files should be called `QuinticWalk.yaml` not `Quinticwalk.yaml`.